data/persons.txt

```diff
@@ -405,3 +353,181 @@ person "Tranquility"
 			"I swear by my pretty floral bonnet, I will end you."
 			"Just once I'd like things to go according to the gorram plan."
 			"I'm a dangerous-minded man on a ship loaded with hurt."
+
+
+
+person "Power of the People"
+	government "Author"
+	frequency 200
+	personality
+		mining harvests opportunistic frugal unconstrained
+	phrase
+		phrase
+			"friendly author"
+	phrase
+		word
+			"Does it sound like I'm ordering a pizza?"
+			"I've made a lot of special modifications myself."
+			"I take orders from just one person: me."
```

Comment:
I feel like "myself" would work better here.

Reply:
https://society6.com/product/han-solo-i-take-orders-from-just-one-person-me_print

Reply:
And *of course* it's a reference.

Reply:
It's a Star Wars reference; you'll have to take it up with Han Solo. 🙂